modules/gapi/test/stereo/gapi_stereo_tests.cpp

```diff
@@ -0,0 +1,103 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include "../test_precomp.hpp"
+
+#include <opencv2/calib3d.hpp>
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+
+namespace opencv_test {
+
+namespace
+{
+#define STR_CPU [] () { return cv::compile_args(cv::gapi::use_only{cv::gapi::str::cpu::kernels()}); }
```

Comment:
How is it called in another tests ?

Reply:
I did a parameterized tests
Now it is used there